src/prodbg/viewdock/src/area/split.rs

```diff
@@ -119,18 +118,25 @@ impl Split {
             self.ratios[index + 1] - 0.05
         };
 
-        if res < min {
-            res = min;
+        if ratio < min {
+            ratio = min;
```

Comment:
Yeah I looked for clamp but couldn't find one :/ but maybe you have better luck :)